Ideas-2022.md

```diff
@@ -2,7 +2,7 @@
 
 ## Project Ideas
 * [Improve and maintain 12 Sugar activities](#improve-and-maintain-12-sugar-activities)
-* [Improve and maintain 6 Sugar activities](#improve-and-maintain-6-sugar-activities)
+* [Improve and maintain 8 Sugar activities](#improve-and-maintain-6-sugar-activities)
```

Comment:
There's another 6 on this line to change.

Reply:
Thanks for pointing that out.